### ISSUE TITLE
Make seperate categories for games on multiple platforms

### ISSUE THREAD

**Issue by matty45** [HaloBible/wiki#27]
https://i.imgur.com/4mC8GSd.png

**Comment by no1dead** [comment]
Had tried this previously, but since it would be easier, and a lot nicer to look at if it was.

"game>platform (if needed)>article."
instead of 
"platform>games>article

Plus it removes a lot of duplication in folders.

**Comment by MegaMech** [comment]
I agree with no1 dead. This is fine like this. If need be we could have a sub-article on one or the other, foot notes, or a hide/show button so reader can choose the right steps  within a guide

**Comment by MegaMech** [comment]
Matty do you have discord? You can message me if you have questions about things. Presumably, you can message no1dead too, but I don't want to speak for him. My discord name is the same as here.

**Comment by matty45** [comment]
i already pinged you about a thing yesterday

**Comment by MegaMech** [comment]
Ah, ya I probably should have pinged you in my reply to that.

**Comment by PaleLebouf** [comment]
I will add this change soon. I think it'd be a great idea to seperate by specific game category/platform if we also want to expand and bring in knowledge for the 360 version of games or previous knowledge of modding Halo PC/CE.

**Comment by MegaMech** [comment]
PaleLebouf, do you have Discord? You should probably chat with No1Dead before making this change.

Also, view epitaph's repo history to see what it looked like before. The layout is much better this way than before.

Instead, if something only applies to xbox or PC we can add tabs or footnotes in an article that describes the correct method for a platform. This resolves the issue of having the same content. Plus nobody else has contributed enough content for platform related issues to exist.
